recipe/meta.yaml

```diff
@@ -18,7 +21,7 @@ outputs:
             - __osx >={{ MACOSX_DEPLOYMENT_TARGET|default("10.12") }}  # [osx and x86_64]
     requirements:
       run:
-        - rust >={{ version }}
+        - rust >{{ lower_bound_version }},<{{ version_stable }}
```

Comment:
Wouldn't be `={{version_stable}}` better?

Reply:
It wouldn't work because it would get rust 1.76, which doesn't have features from the dev builds.